docs/releases/3.0_workingdraft/dcat-ap-ch_3.0_workingdraft.html

```diff
@@ -4709,7 +5377,7 @@ <h2>Verzeichnis der referenzierten Namensräume</h2>
       <tbody>
       <tr>
         <td><code>dcat</code></td>
-        <td><code>http://www.w3.org/ns/dcat#</code></td>
+        <td><code>https://www.w3.org/TR/vocab-dcat-3/</code></td>
```

Comment:
```suggestion
        <td><code>http://www.w3.org/ns/dcat#</code></td>
```
The namespace has not changed even though DCAT is now at version 3 (see https://www.w3.org/TR/vocab-dcat-3/#namespaces).